engine/flink/management/src/main/scala/pl/touk/nussknacker/engine/management/FlinkDeploymentManager.scala

```diff
@@ -58,7 +59,7 @@ abstract class FlinkDeploymentManager(modelData: BaseModelData, shouldVerifyBefo
       oldJob <- OptionT(checkOldJobStatus(processVersion))
       deploymentId <- OptionT.fromOption[Future](oldJob.deploymentId)
       //when it's failed we don't need savepoint...
-      if oldJob.status.isRunning || oldJob.status.isDuringDeploy
+      if Set(SimpleStateStatus.Running, SimpleStateStatus.DuringDeploy).contains(oldJob.status)
```

Comment:
You declare this Set in the serval places. Let's move that to SimpleStateStatus.RunningStates = Set(SimpleStateStatus.Running, SimpleStateStatus.DuringDeploy). It provides an easier way to add/remove an option.

Reply:
I'll extract constant but only when you give me a good name for it. DuringDeploy certainly doesn't mean that scenario is running. Try again :-P

Reply:
ActiveStatutes? :)

Reply:
In fact, you already named it: isFollowingDeployStatus ? :)

Reply:
You won